calibnet-config.toml

```diff
@@ -1,26 +1,6 @@
-[ado]
```

Comment:
we're we not going to keep this in?

Reply:
It will still get picked up if we are debugging but end users won't need to set this, that was the thinking behind removing it.